roles/keystone-setup/tasks/main.yml

```diff
@@ -42,6 +42,7 @@
                  tenant={{ item.tenant }}
                  token={{ admin_token }}
   with_items: keystone.users
+  when: "{{ item.name }}.enabled is undefined or {{item.name}}.enabled|bool"
```

Comment:
I'm not sure I understand. If you're looping through keystone.users, there will always be an item, even if it's empty. So item exists, item.enabled doesn't, the default applies to the enabled bit. Maybe you could show screen output where you're seeing something different?